casper/src/main/scala/coop/rchain/casper/genesis/Genesis.scala

```diff
@@ -292,8 +291,8 @@ object Genesis {
         Sync[F].delay {
           keys.foreach { //create files showing the secret key for each public key
             case (sec, pub) =>
-              val sk      = Base16.encode(sec)
-              val pk      = Base16.encode(pub)
+              val sk      = Base16.encode(sec.bytes)
```

Comment:
Not now but at some point later we should hide the bytes and allow only access via some API like fromBase16/toBase16/sign/verify.